server/pages/index.js

```diff
@@ -359,7 +365,24 @@ module.exports = function() {
 			// the user is probably logged in
 			renderLoggedInRoute( req, res );
 		} else {
-			renderLoggedOutRoute( req, res );
+			const context = getDefaultContext( req );
+
+			try {
+				context.layout = ReactDomServer.renderToString( LayoutLoggedOutDesignFactory() );
+			} catch ( ex ) {
+				if ( config( 'env' ) === 'development' ) {
+					throw ex;
+				}
+			}
+
+			const {
+				title: helmetTitle,
+				meta: helmetMeta,
+				link: helmetLink
+			} = Helmet.rewind();
+			Object.assign( context, { helmetTitle, helmetMeta, helmetLink } );
```

Comment:
Here, I chose to rename the properties rather than:
1. keeping the names and merging into `context`
2. keeping the names and merging into `context.head`

because
1. I wanted to avoid any potential naming conflicts or any confusion
2. I wanted to avoid the verbosity of `if head && head.meta` in Jade

**but** I can go back to any of those two alternatives. Let me know.